Code/EnginePlugins/AngelScriptPlugin/TypeRegistration/AsRegisterTypes_String.cpp

```diff
@@ -130,8 +135,8 @@ void ezAngelScriptEngineSingleton::Register_StringView()
   AS_CHECK(m_pEngine->RegisterObjectMethod("ezStringView", "bool TrimWordStart(ezStringView sWord)", asMETHOD(ezStringView, TrimWordStart), asCALL_THISCALL));
   AS_CHECK(m_pEngine->RegisterObjectMethod("ezStringView", "bool TrimWordEnd(ezStringView sWord)", asMETHOD(ezStringView, TrimWordEnd), asCALL_THISCALL));
 
-  AS_CHECK(m_pEngine->RegisterObjectMethod("ezStringView", "bool opEquals(ezStringView) const", asFUNCTIONPR(operator==, (ezStringView, ezStringView), bool), asCALL_CDECL_OBJFIRST));
-  AS_CHECK(m_pEngine->RegisterObjectMethod("ezStringView", "int opCmp(ezStringView) const", asFUNCTIONPR(ezStringView_opCmp, (ezStringView, ezStringView), int), asCALL_CDECL_OBJFIRST));
+  AS_CHECK(m_pEngine->RegisterObjectMethod("ezStringView", "bool opEquals(const ezStringView& in)", asFUNCTIONPR(ezStringView_opEqual, (ezStringView*, const ezStringView&), bool), asCALL_CDECL_OBJFIRST));
```

Comment:
opEquals should probably still be const

Reply:
Oh right! Thanks for noticing. I'm still getting a hang of angel script function declarations.